worker/charmrevision/validate_test.go

```diff
@@ -28,6 +29,7 @@ func (s *ValidateSuite) SetUpTest(c *gc.C) {
 		RevisionUpdater: struct{ charmrevision.RevisionUpdater }{},
 		Clock:           struct{ clock.Clock }{},
 		Period:          time.Hour,
+		Logger:          loggo.GetLogger("test"),
```

Comment:
There is usually a fakeLogger{} around to use instead.  Or you can make one

Reply:
Interestingly, I had a little fake logger (`blackholeLogger`) in an earlier commit, and Ian said we usually use `GetLogger("test")` so I changed it to that. :-)

Reply:
I was just going for consistency with what most other tests did. If we want to use a fake or no-op logger, perhaps we can add one to juju/testing somewhere.